README.md

```diff
@@ -18,16 +18,18 @@ If your ECIP requires images, the image files should be included in a subdirecto
 When you believe your ECIP is mature and ready to progress past the draft phase, you should do one of two things:
 
  - **For a Standards Track ECIP of type Core**, ask to have your issue added to the agenda of an upcoming All Dev Team meeting, where it can be discussed for inclusion in a future hard fork. If implementers agree to include it, the ECIP editors will update the state of your ECIP to 'Accepted'.
- - **For all other ECIPs**, open a PR changing the state of your ECIP to 'Final'. An editor will review your draft and ask if anyone objects to its being finalised. If the editor decides there is no rough consensus - for instance, because contributors point out significant issues with the ECIP - they may close the PR and request that you fix the issues in the draft before trying again.
+ - **For all other ECIPs**, open a PR changing the state of your ECIP to 'Final'. An editor will review your draft and ask if anyone objects to it being finalized. If the editor decides there is no rough consensus - for instance, because contributors point out significant issues with the ECIP - they may close the PR and request that you fix the issues in the draft before trying again.
 
 # ECIP Status Terms
-* **Draft** - an ECIP that is undergoing rapid iteration and changes
-* **Last Call** - an ECIP that is done with its initial iteration and ready for review by a wide audience
-* **Final (non-Core)** - an ECIP that has been in Last Call for at least 2 weeks and any technical changes that were requested have been addressed by the author.
-* **Final (Core)** - an ECIP that the dev teams have decide to implement and release in a future hard fork or has already been released in a hard fork
-* **Accepted** - a core ECIP that has been in Last Call for at least 2 weeks and any technical changes that were requested have been addressed by the author
-
-* **Deferred** - an ECIP that is not being considered for immediate adoption. May be reconsidered in the future for a subsequent hard fork.
+* **Draft** - An ECIP was submitted by its author and is undergoing rapid iteration and changes.
+* **Last Call** - An ECIP that is done with its initial iteration and ready for review by a wider audience.
+* **Accepted** - An ECIP that has been in Last Call for at least 2 weeks and any technical changes that were requested have been addressed by the author. If Core Devs decide to implement an ECIP into their clients, the ECIP will move to final.
+* **Final (non-Core)** - An ECIP that has been in Last Call for at least 2 weeks, any technical changes that were requested have been addressed by the author, and Core Devs decided to implement it into their clients. 
```

Comment:
That reads as if a final ECIP can not have any implementation, which is technically incorrect. Final should always mark ECIPs that are beyond accepted, implemented, and in a state where it can no longer be modified.

Reply:
For some reason both Ethereum EIP and Ethereum Classic ECIP Status Terms have divided the "Final" status into two" Final (non-core) and Final (Core). 

It seems there is and intermediate period between "Accepted" and "Final"  where devs implement changes, but are not scheduled or deployed in a hard fork.

If that intermediate period is true or really exists, then we can change the term Final (non-Core) to "Intermediate" or "Pre-final".

If not we can strike "Final (non-Core)" altogether.

Please let me know if that intermediate state actually exists or what terminology you would use.

Reply:
Yeah, I think we should simplify it here.

Draft > Last Call > Accepted > Final; I don't see how non-core proposals should follow a different route.

Reply:
Done.

Also I unified both "final" sections and used the text you suggested below.